pom.xml

```diff
@@ -276,7 +276,8 @@
                 <configuration>
                     <header>com/mycila/maven/plugin/license/templates/MIT.txt</header>
                     <properties>
-                        <owner>DTL</owner>
+                        <owner>FAIR Data Team</owner>
+                        <year>2024</year>
```

Comment:
I'm not sure what a good practice is, but do we need a range here? like `2016-2024` and keep that updated every year?